### PR TITLE
42 docker and distributed

### PR DESCRIPTION
**Pull request by timjon**

### CODE REVIEW

File: Application/src/view/visuals/Draw.java [hunk 1]
```diff
@@ -138,7 +121,8 @@ public void addClassDiagramClass (String name){
      * @param device
      */
     public void addDeploymentDiagramClass (String device){
-        allDeploymentClasses.add(new DeploymentDiagramClass(device));
+        if (!allClassDiagramClasses.contains(device))
+            allDeploymentClasses.add(new DeploymentDiagramClass(device));
```
Comment (kosara-g) [inline Application/src/view/visuals/Draw.java:125]: wait, is this hardcoded?

**Comment by kosara-g** [comment]
seems to have conflicts with develop

File: Application/src/view/visuals/Draw.java (hunk 1 above)
Comment (pontuslaestadius) [reply Application/src/view/visuals/Draw.java:125]: no? This was added in 57, which is still open.

File: Application/src/controller/network/Decode.java [hunk 2]
```diff
@@ -12,7 +12,7 @@
 /**
  * @author Pontus Laestadius
  * Collaborators: Sebastian Fransson, Tim Jonasson, Kosara Golemshinska
- * @version 1.5
+ * @version 1.6
```
Comment (IsabelleTornqvist) [inline Application/src/controller/network/Decode.java:15]: Version

File: Server/src/node.erl [hunk 3]
```diff
@@ -12,56 +12,27 @@ init(Coordinator, Name) ->
 loop(Coordinator, Name, Class) ->
   receive
   
-  % Gets the name of the node.
-  {Pid, getName} -> 
-  	Pid ! {getName, Name},
-  	loop(Coordinator, Name, Class);
+    % Gets the name of the node.
```
Comment (IsabelleTornqvist) [inline Server/src/node.erl:15]: Remove if not used? Shouldn't be on develop

File: Server/src/lobbycoordinator.erl [hunk 4]
```diff
@@ -73,6 +73,7 @@ loop(Rooms, Lobby_increment) ->
 	  
 	%This happens when you send a new diagram to the lobby
 	{Creator_Socket, {Did, Class_names, Classes, Messages}} -> 
+	  io:format("Im creating stuff ~n"),
```
Comment (IsabelleTornqvist) [inline Server/src/lobbycoordinator.erl:76]: Remove?

**Comment by pontuslaestadius** [comment]
Included in Demo

**Pull request closed without merging**